src/gateio_new_coins_announcements_bot/logger.py

```diff
@@ -7,7 +7,7 @@
 from gateio_new_coins_announcements_bot.send_telegram import TelegramLogFilter
 
 # loads local configuration
-config = load_config("config.yml")
+config = load_config("src/config.yml")
```

Comment:
I wasn't sure if this was necessary, but it fixed some compiler warnings for me.

Reply:
Fixed all these